src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/cross-app-tracing.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: "Configure cross-application tracing"
+type: apiDoc
+shortDescription: Manually link transactions between your mobile and APM-monitored applications.
+tags:
+  - Mobile monitoring
+  - New Relic Mobile Android
+  - Android SDK API
+metaDescription: Mobile app monitoring API used to manually link transactions between your mobile and APM-monitored applications.
+redirects:
+---
+
+## Manual cross application tracing [#manual-cat]
```

Comment:
This h2 looks a little clunky right next to the page title. Could it be preceded by some of the intro text?/ Is it necessary enough to include that it should replace the main title?

Reply:
Also, if this process is Android-specific, could that be made a bit clearer?